website/docs/r/media_store_container.html.markdown

```diff
@@ -0,0 +1,33 @@
+---
+layout: "aws"
+page_title: "AWS: aws_media_store_container"
+sidebar_current: "docs-aws-resource-media-store-container"
+description: |-
+  Provides a MediaStore Container.
+---
+
+# aws_media_store_container
+
+Provides a MediaStore Container.
+
+## Example Usage
+
+```hcl
+resource "aws_media_store_container" "example" {
+  name = "example"
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `name` - (Required) The name of the container. Must atisfy regular expression pattern: `\w+`
```

Comment:
Can we make this a little bit more human-readable for folks who don't know regular expressions or omit this sentence (as the validation error message will tell them anyways)?